### DISCUSSION TITLE
157

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#99] [General]
# 潮流周刊第157期 - 引力剧场

封面图来源于上上周末在上海玩的时候，刚好浦东美术馆有几个展晚上就过去看了看，这个徐冰的引力剧场实在太棒了，去上海玩的小伙伴可以去瞧瞧。

https://weekly.tw93.fun/posts/157-%E5%BC%95%E5%8A%9B%E5%89%A7%E5%9C%BA

**Comment by baixiang-main** [comment]
浦东美术馆确实挺好的，毕竟一价全包，满展的话超值

**Reply by tw93** [reply]
是的 我感觉挺好的

**Comment by cuncinc** [comment]
是[az].com，还是[a-z].com

**Reply by tw93** [reply]
`-` 应该是被markdown转义了
